src/pipeline/src/etl/transform/transformer/greptime.rs

```diff
@@ -38,6 +41,8 @@ use crate::etl::transform::{Transform, Transformer, Transforms};
 use crate::etl::value::{Timestamp, Value};
 
 const DEFAULT_GREPTIME_TIMESTAMP_COLUMN: &str = "greptime_timestamp";
+const DEFAULT_MAX_NESTED_LEVELS_FOR_JSON_FLATTENING: usize = 10;
+const GREPTIME_IDENTITY_PIPELINE_PARAMS_HEADER: &str = "x-greptime-identity-pipeline-params";
```

Comment:
Why limit it to just the identity pipeline? Other pipelines can use these parameters too. The query params can simply be a string map.
We won't be able to add more headers in the future. It makes more sense to have all pipelines share a single header for custom parameters.